src/ProcessManager/ChromeManager.php

```diff
@@ -89,4 +92,14 @@ private function getDefaultArguments(): array
 
         return $args;
     }
+
+    private function getDefaultOptions(): array
+    {
+        return [
+            'https' => false,
```

Comment:
`'scheme' => 'https' ` to respect the format of [`parse_url`](http://php.net/manual/en/function.parse-url.php) and to allow using potential exotic schemes?

Reply:
Way better than what I had... :D